### PR TITLE
Zippopotamus (Independent Publisher)

### PR DESCRIPTION
**Pull request by tposzytek**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [X] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

Connector validation!

Test flow validation!

CLI validation!

### CODE REVIEW

**Comment by natalie-pienkowska** [comment]
Thanks, @tposzytek , for your connector submission! (I see it's a Geolocation type of connector). This week is Thanksgiving, so there may be a delay in looking at your PRs until 11/29. Hope you have a great week :)

File: independent-publisher-connectors/Zippopotamus/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,375 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Zippopotamus",
+    "description": "API allows to obtain location details for a given postal code or state and city. It provides data for over 60 countries.",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Zippopotamus/apiDefinition.swagger.json:5]: @tposzytek Can you please add the product/service details to the description as it seems API centric explanation?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/Zippopotamus/apiDefinition.swagger.json (hunk 1 above)
Comment (tposzytek) [reply independent-publisher-connectors/Zippopotamus/apiDefinition.swagger.json:5]: Is this somehow closer to what you were expecting?

**Comment by sriyen-msft** [comment]
@tposzytek Apologies for the delay in response, We have a high number of submissions and had to prioritize the connectors which were ready for production deployment. 

We are preparing your connector for the next deployment schedule that is on Monday 3rd Jan 2022. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes between 7-10 business days. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps. Thanks.

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@tposzytek Apologies for the delay in response.
Update - Your connector is now in production deployment and reached [Test](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#region-deployment) region, and it would be completed and available in all the production regions within 3 weeks. Please let us know if you have any questions, Thanks.

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@tposzytek Apologies for the delay in response. 

Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Pull request merged**